internal.nix

```diff
@@ -294,14 +294,20 @@ rec {
         else version);
       dependencies = if (content ? dependencies) then lib.mapAttrs patchDep content.dependencies else { };
       devDependencies = if (content ? devDependencies) then lib.mapAttrs patchDep content.devDependencies else { };
+      defaultScripts = if (content ? scripts) then content.scripts else { };
+      scripts = defaultScripts // {
+        preinstall = "${preinstallScript};" + (if (defaultScripts ? preinstall) then defaultScripts.preinstall else "");
```

Comment:
```suggestion
        preinstall = "${preinstallScript};" + (lib.optionalString (defaultScripts ? preinstall) defaultScripts.preinstall);
```

Reply:
Ahh, I will need to think about this a little bit more. Worked for my workflow but still had to override single package with patchesShebangs.